src/Main.java

```diff
@@ -5,10 +5,10 @@ public static void main(String[] args) throws InterruptedException {
         System.out.println("The clock is starting");
         for (int i = 0; i <= 100 ; i++) {
             System.out.println("TIC");
-            Thread.sleep(1000);
+            Thread.sleep(3000);
             System.out.println("SI");
             System.out.println("TAC");
-            Thread.sleep(1000);
+            Thread.sleep(3000);
```

Comment:
you should get the value from the clock object inside the person object.
Not hardcode 3 seconds.
Something like 
Thread.sleep(person.getClock().getInterval() * 1000);